dvc/commands/ls/__init__.py

```diff
@@ -123,6 +141,7 @@ def add_parser(subparsers, parent_parser):
         ),
     )
     list_parser.add_argument("--size", action="store_true", help="Show sizes.")
+    list_parser.add_argument("--md5", action="store_true", help="Show MD5 checksums.")
```

Comment:
`dvc` also supports other hashes than md5 (which is `md5-dos2unix`, a legacy md5 hash used in 2.x that is still supported). And internally, dvc is agnostic to a specific hash.

Also, we have a `--show-hash` command in `dvc diff`, so I think we should use the same terminology here. 

 
```suggestion
    list_parser.add_argument(
        "--show-hash",
        help="Display hash value for each item",
        action="store_true",
        default=False,
    )
```